gnocchi/tests/functional/gabbits/base.yaml

```diff
@@ -15,6 +15,7 @@ tests:
   response_headers:
       content-type: /^application\/json/
   response_json_paths:
+      $.build[0]: "5"
```

Comment:
does this mean we have to change this every single time we tag?

Reply:
Yes, at least each time we change the major version pbr thinks we're going to release.

@sileht what bothers me is that pbr thinks we'll release 5 (which I can tblame it for) but when we'll tag 4.1.0 this test will fail for that particular tag.

Reply:
:( seems like a pain in the ass... but i have no suggestion so meh.

i thought pbr appends devXYZ to the end of master builds where XYZ is changing all the time?